src/main/java/org/broadinstitute/hellbender/utils/locusiterator/LocusIteratorByState.java

```diff
@@ -352,6 +445,37 @@ private void lazyLoadNextAlignmentContext() {
     }
 
     /**
+     * Fix the quality of two elements that comes from an overlaping pair
```

Comment:
Please provide documentation what is being done here. Something more specific than 'fix the quality' - what's the exact algorithm?